buildSrc/src/main/groovy/com/uber/okbuck/rule/AndroidRule.groovy

```diff
@@ -141,9 +147,11 @@ abstract class AndroidRule extends BuckRule {
         printer.println("\tsource = '${mSourceCompatibility}',")
         printer.println("\ttarget = '${mTargetCompatibility}',")
         if (!mPostprocessClassesCommands.empty) {
+            String deps = "\$(JARS=(`find ${mGenDir} ! -name \"*-abi.jar\" ! -name \"*dex.dex.jar\" -name \"*.jar\"`); IFS=:; echo \"\${JARS[*]}\")"
+            String androidJar = mBootClasspath
             printer.println("\tpostprocess_classes_commands = [")
             mPostprocessClassesCommands.each { String command ->
-                printer.println("\t\t'${command}',")
+                printer.println("\t\t'export DEPS=${deps}; export ANDROID_JAR=${androidJar}; ${command}',")
```

Comment:
we do not want to export these. These should just be inlined with the command to be invoked